bridge/src/lib.rs

```diff
@@ -320,7 +320,8 @@ pub mod pallet {
 			// Extract dest (MultiLocation) to get corresponding Ethereum recipient address
 			let recipient = T::ExtractRecipient::extract_recipient(&dest)
 				.ok_or(Error::<T>::ExtractRecipientFailed)?;
-			let fee = T::FeeHandler::get_fee(&asset.id).ok_or(Error::<T>::MissingFeeConfig)?;
+			let fee = T::FeeHandler::get_fee(T::DestDomainID::get(), &asset.id)
```

Comment:
should here be calling feeHandlerRouter and then call the `get_fee` on the router?

Reply:
yes that is what I meant, because in the runtime/lib.rs, it was `type FeeHandler = SygmaBasicFeeHandler;`, but now you changed it to be `type FeeHandler = FeeHandlerRouter;`, so that `T::FeeHandler::get_fee` now is the `FeeRouter::get_fee`, it all makes sense now.